hardware/swabian_instruments/pulse_streamer.py

```diff
@@ -153,120 +208,174 @@ def get_constraints(self):
         # channels. Here all possible channel configurations are stated, where only the generic
         # names should be used. The names for the different configurations can be customary chosen.
         activation_config = OrderedDict()
-        activation_config['all'] = ['d_ch1', 'd_ch2', 'd_ch3', 'd_ch4', 'd_ch5', 'd_ch6', 'd_ch7',
-                                    'd_ch8']
+        activation_config['all'] = {'d_ch1', 'd_ch2', 'd_ch3', 'd_ch4', 'd_ch5', 'd_ch6', 'd_ch7',
```

Comment:
According to the pulser_interface this set should be of type `frozenset`

Reply:
didn't realise that before, is that new? fixed

Reply:
Thanks.
Yes, that was introduced a bit after pulsed 3.0.
Since these configurations should be fixed sets of channels it does not make sense for them to be mutable.